.github/actions/status-reporter/action.yml

```
@@ -0,0 +1,84 @@
+        # Save report to file for GitHub output
+        echo "$REPORT" > status_report.md
+
+        # Set GitHub Action outputs
+        {
+          echo "status-table<<EOF"
+          echo "$REPORT"
+          echo "EOF"
+        } >> $GITHUB_OUTPUT
+
+        {
+          echo "summary-report<<EOF" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)